aten/src/ATen/native/native_functions.yaml

```diff
@@ -452,6 +452,42 @@
     CPU: argmin
     CUDA: argmin
 
+- func: arccosh(Tensor self) -> Tensor
```

Comment:
Why is the name `arccosh` here and not `acosh`? Given that the current function we have is `acos` ? (same for the others).

Reply:
I thought the same but then, NumPy uses `arccosh` (though, it also uses `arccos`) - so I decided to go for `arccosh` unless and until `acosh` is suggested. Do you suggest using `acosh`, `asinh` and `atanh` instead?

Reply:
I guess @mruberry will give the last call here.
I would vote for staying consistent within pytorch.
But if we say, acos is going to be deprecated in favor of arccos in the future. I am happy with it and going directly for arccosh here.

Reply:
In the future we'll support both (via aliasing). Let's go acosh for now for consistency (aliasing will come later).

Reply:
Thanks @albanD and @mruberry! I've changed everything to `acosh` format. (https://github.com/pytorch/pytorch/pull/38388/commits/7df086c1a36fdef1f3b7badb76fa1812b83a14f7)